pkg/poseidonhttp/poseidonhttp.go

```diff
@@ -58,7 +58,11 @@ func newHealthzHandler(hfunc func() Health) http.HandlerFunc {
 			return
 		}
 		h := hfunc()
-		d, _ := json.Marshal(h)
+		d, err := json.Marshal(h)
+		if err != nil {
+			glog.Errorf("marshal failed, err: %v", err)
```

Comment:
Nit: Log message should be started with upper-case word, e.g. Failed to marshal %v, error: %v

Reply:
OK, I updated it.